src/include/wiredtiger.in

```diff
@@ -4773,66 +4748,49 @@ struct __wt_location_handle {
  */
 struct __wt_storage_source {
 	/*!
-	 * Return a location handle from a location string.
-	 * A location string may encode a bucket name, or the equivalent for this
-	 * storage source, authorization information for that bucket,
-	 * naming prefixes to be used for objects in that bucket, etc.
-	 *
-	 * @errors
-	 *
-	 * @param storage_source the WT_STORAGE_SOURCE
-	 * @param session the current WiredTiger session
-	 * @param location the location string
-	 * @param[out] location_handle the allocated handle
-	 */
-	int (*ss_location_handle)(WT_STORAGE_SOURCE *storage_source,
-	    WT_SESSION *session, const char *location, WT_LOCATION_HANDLE **location_handle);
-
-	/*!
-	 * Return a list of object names for the given location.
-	 *
-	 * @errors
-	 *
-	 * @param storage_source the WT_STORAGE_SOURCE
-	 * @param session the current WiredTiger session
-	 * @param location_handle the location to list
-	 * @param prefix if not NULL, only files with names matching the prefix
-	 *    are returned
-	 * @param limit if not 0, limits the number of objects listed to this number.
-	 * @param[out] object_list the method returns an allocated array of
-	 *    individually allocated strings, one for each object in the location.
-	 * @param[out] countp the number of entries returned
-	 */
-	int (*ss_location_list)(WT_STORAGE_SOURCE *storage_source,
-	    WT_SESSION *session, WT_LOCATION_HANDLE *location_handle, const char *prefix,
-	    uint32_t limit, char ***object_list, uint32_t *countp);
-
-	/*!
-	 * Free memory allocated by WT_STORAGE_SOURCE::location_list.
-	 *
-	 * @errors
-	 *
-	 * @param storage_source the WT_STORAGE_SOURCE
-	 * @param session the current WiredTiger session
-	 * @param object_list array returned by WT_STORAGE_SOURCE::location_list
-	 * @param count count returned by WT_STORAGE_SOURCE::location_list
-	 */
-	int (*ss_location_list_free)(WT_STORAGE_SOURCE *storage_source,
-	    WT_SESSION *session, char **object_list, uint32_t count);
-
-	/*!
-	 * Return if the named object exists in the location.
-	 *
+	 * Create a customized file system to access the storage source
+         * objects.
```

Comment:
There is some tab/space formatting issue going on here I think.

Reply:
Fixed.